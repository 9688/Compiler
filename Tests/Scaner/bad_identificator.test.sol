Ошибка в строке 1 позиции 6: требуется экспоненциальное значение
IDENTIFICATOR 1 7 wrw
EOF 1 10 
