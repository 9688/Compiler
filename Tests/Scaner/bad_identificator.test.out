Ошибка в строке 1 позиции 6: требуется экспоненциальное значение
EOF 1 10 
